integrations/jenkins/documentation.yaml

```diff
@@ -0,0 +1,77 @@
+exporter_type: included
+app_name_short: Jenkins
+app_name: {{app_name_short}}
+app_site_name: Jenkins
+app_site_url: https://www.jenkins.io/
+exporter_name: the Jenkins exporter
+exporter_pkg_name: jenkins
+exporter_repo_url: https://github.com/jenkinsci/prometheus-plugin
+additional_prereq_info: |
+  {{exporter_name}} exposes Prometheus-format metrics when the [metrics](https://plugins.jenkins.io/metrics/)
+  and [prometheus](https://plugins.jenkins.io/prometheus/) plugins are installed.
+  You can run the following checks to verify that {{app_name_short}} is exposing Prometheus-format metrics.
+
+   *  To determine if {{exporter_name}} is exposing metrics, run the following command:
```

Comment:
Same comment as in https://github.com/GoogleCloudPlatform/monitoring-dashboard-samples/pull/423#discussion_r1063786149

Reply:
Concur!

Reply:
Sorry for the churn... same comment as Velero about numbering?